### ISSUE TITLE
Fix build status inconsist

### ISSUE THREAD

**Issue by pzou1974** [intuit/foremast-brain#63]
https://www.travis-ci.org/intuit/foremast-brain status is success but https://github.com/intuit/foremast-brain readme status is failed

this appears on dev branch.

**Comment by shoshonte** [comment]
this is an issue with the travis-ci plugin... so you would need to contribute to travis-ci to fix this discrepancy.

**Comment by rlcooper46** [comment]
Seems that master branch does not have the python test file... it looks like it was removed in the last master commit: e1e0058b84bd04471f37cb790c13d7322aef1875. I have added it in PR #64

**Comment by tekgal** [comment]
fixed it as part of https://github.com/intuit/foremast-brain/pull/65
Sample pre-commit output
```
git add .
git commit -m "add prehook to make travis build status branch specific"
Starting pre-commit hook...
Replacing:
	[![Build Status](https://api.travis-ci.org/intuit/foremast_brain.png?branch=dev)]

with:
	[![Build Status](https://api.travis-ci.org/intuit/foremast_brain.png?branch=test)]

pre-commit hook complete.
```

**Comment by tekgal** [comment]
fixed it as part of #65
Sample pre-commit output
```
git add .
git commit -m "add prehook to make travis build status branch specific"
Starting pre-commit hook...
Replacing:
	[![Build Status](https://api.travis-ci.org/intuit/foremast_brain.png?branch=dev)]

with:
	[![Build Status](https://api.travis-ci.org/intuit/foremast_brain.png?branch=test)]

pre-commit hook complete.

```

**Comment by tekgal** [comment]
> fixed it as part of #65
> Sample pre-commit output
> 
> ```
> git add .
> git commit -m "add prehook to make travis build status branch specific"
> Starting pre-commit hook...
> Replacing:
> 	[![Build Status](https://api.travis-ci.org/intuit/foremast_brain.png?branch=dev)]
> 
> with:
> 	[![Build Status](https://api.travis-ci.org/intuit/foremast_brain.png?branch=test)]
> 
> pre-commit hook complete.
> ```

on further analysis this would not work because there was not a straight-forward way to ensure README.md update on PR merge.
on further testing i validated that removing the branch name form the Readme might fix the issue as seen in https://github.com/tekgal/foremast-brain/tree/test and https://github.com/tekgal/foremast-brain/tree/test-new
